bleach/callbacks.py

```diff
@@ -18,6 +22,10 @@ def nofollow(attrs, new=False):
 
 def target_blank(attrs, new=False):
     href_key = (None, u'href')
+
+    if href_key not in attrs:
+        return attrs
```

Comment:
Mmm... It looks like neither of these callbacks have any tests. That's terrible. Sorry about that. I'll throw some together after landing this.